Fixes layouts

kchobantonov opened the pull request:
Fixing layouts, translations, minor fixes

kchobantonov commented:
@sdirix  @lucas-koehler  please review

kchobantonov commented:
@sdirix  will you have some time next week to review this and if good to have a release ? Also did you decide if we want to include the FileRenderer and perhaps close the PR for the TemplateRenderer if indeed you do not want that added to this project like the webcomponent.

lucas-koehler commented:
Hello @kchobantonov , sorry for the late reply and thanks again for the contributions. We are going to review this PR and create a release in the next two weeks. We decided to not introduce the template renderer. I'll close that PR.

sdirix commented on vue2-vuetify/src/complex/ArrayControlRenderer.vue:
```diff
@@ -110,7 +114,7 @@
                       </v-btn>
                     </template>
-                    Move Up
+                    {{ t('array.btn.moveUp.tooltip', 'Move Up') }}
```
We just merged a PR on the main repository which provides these translations: https://github.com/eclipsesource/jsonforms/pull/2129
We'll release on the main repository next week, then we can update this PR to consume these translations.

kchobantonov replied:
@sdirix  I have changed most of the translations but for the up/down tooltip we are missing the properties - we have only the up/down for the aria label - or should I use that for now ?

sdirix replied:
good point. let's keep it like this for now

sdirix commented on vue2-vuetify/src/complex/OneOfRenderer.vue:
```diff
@@ -42,18 +42,28 @@
       <v-card>
-        <v-card-title class="text-h5"> Clear form? </v-card-title>
+        <v-card-title class="text-h5">
+          {{ t('form.clear.title', 'Clear form?') }}
```
This should ideally then work in a similar way like the mentioned PR above.

kchobantonov replied:
I couldn't find the properties for that in the jsonforms core yet or maybe I'm not looking at the right place - can you point me to the locations in the jsonforms core where this is defined the same way how we have the arrayTranslations ?

sdirix replied:
The idea would be to handle this via the binding and pass down the translated labels instead of calling the translator within the renderer. But for now I'm fine with this.

sdirix reviewed (commented):
Looks good in general! I would like to suggest to refactor this PR next week as suggested when the next prerelease of JSON Forms is published.

kchobantonov commented:
@sdirix  I have updated this branch to use jsonforms 3.1.0-alpha.3 and I added the suggested changes for the translations except for the tooltips for 


array.btn.moveUp.tooltip
array.btn.moveDown.tooltip

I couldn't find any any translation properties for the up/down tooltip - only for the aria label

kchobantonov commented:
@lucas-koehler @sdirix  can you please review - I have included the changes from https://github.com/eclipsesource/jsonforms-vuetify-renderers/pull/92 as well as the suggestions for the translation except for few of those that I couldn't find. Also any plans for a release for this project where this PR is included ? 

Thanks
